.github/workflows/go-update-commenter.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Build full comment
         id: old_versions
         run: |
-          set -euxo pipefail
+          set -euo pipefail
```

Comment:
`set -x` can be pretty useful (for this specific job it helped a lot debugging when something failed) but I understand that it's a security risk so LGTM

Reply:
I think a good practice would be to have a standalone script you can replay in local in the same conditions of the ci and put whatever debugging mode you want, wdyt?